src/Hamiltonians/ExtendedHubbardReal1D.jl

```diff
@@ -42,18 +54,20 @@ Base.getproperty(h::ExtendedHubbardReal1D, ::Val{:add}) = getfield(h, :add)
 Base.getproperty(h::ExtendedHubbardReal1D{<:Any,<:Any,U}, ::Val{:u}) where U = U
 Base.getproperty(h::ExtendedHubbardReal1D{<:Any,<:Any,<:Any,V}, ::Val{:v}) where V = V
 Base.getproperty(h::ExtendedHubbardReal1D{<:Any,<:Any,<:Any,<:Any,T}, ::Val{:t}) where T = T
+Base.getproperty(h::ExtendedHubbardReal1D{<:Any,<:Any,<:Any,<:Any,<:Any,PITWISTED}, ::Val{:pitwisted}) where PITWISTED=PITWISTED
+Base.getproperty(h::ExtendedHubbardReal1D{<:Any,<:Any,<:Any,<:Any,<:Any,<:Any,HARDWALL}, ::Val{:hardwall}) where HARDWALL=HARDWALL
 
 function num_offdiagonals(::ExtendedHubbardReal1D, address::SingleComponentFockAddress)
     return 2 * num_occupied_modes(address)
 end
 
 """
-    extended_bose_hubbard_interaction(address, m)
+    extended_hubbard_interaction(address, m)
 
 Compute the on-site product sum_j n_j(n_j-1) and the next neighbour term
 sum_j n_j n_{j+1} with periodic boundary conditions.
 """
-function extended_bose_hubbard_interaction(b::SingleComponentFockAddress)
+function extended_hubbard_interaction(h::ExtendedHubbardReal1D,b::SingleComponentFockAddress)
```

Comment:
```suggestion
function extended_hubbard_interaction(h::ExtendedHubbardReal1D, b::SingleComponentFockAddress)
```

Reply:
@mtsch had suggested here to add a white space after the comma. While it is a minor issue, it is a reasonable suggestion. Please action it!